conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/TezosNodeOperator.scala

```diff
@@ -65,7 +64,7 @@ class TezosNodeOperator(
     val node: TezosRPCInterface,
     val network: String,
     batchConf: BatchFetchConfiguration,
-    apiOperations: ApiOperations
+    sqlOperations: LorreOperations
```

Comment:
Naming here might be more consistent as `lorreOperations` since the type dependency is explicit.
Seems no use to hide the fact that we have that dependency here with a more generic name.
We might figure out how to better separate the things in future reworks.